devicetypes/smartthings/ikea-button.src/ikea-button.groovy

```diff
@@ -187,7 +188,7 @@ def parse(String description) {
 	} else {
 		if ((description?.startsWith("catchall:")) || (description?.startsWith("read attr -"))) {
 			def descMap = zigbee.parseDescriptionAsMap(description)
-			if (descMap.clusterInt == zigbee.POWER_CONFIGURATION_CLUSTER && descMap.attrInt == 0x0021) {
+			if (!isSomfySituo() && descMap.clusterInt == zigbee.POWER_CONFIGURATION_CLUSTER && descMap.attrInt == 0x0021) {
```

Comment:
I'm not sure if this check: !isSomfySituo() is necessary since You might not receive any report from POWER_CONFIGURATION_CLUSTER.

Reply:
Although the POWER_CONFIGURATION_CLUSTER is not supported it is read when configure() method is called. A response to that command is then received once but it has no value field specified in the descMap so that the NullPointerException occures in the next line: 

event = getBatteryEvent(zigbee.convertHexToInt(descMap.value))

I have added the !isSomfySituo() condition to avoid reaching that line.

Reply:
What do You think about avoiding those calls in configure() method ? That way Somfy devices wouldn't receive those calls in first place and they wouldn't have to respond with no value.

Reply:
Ok, that is a better way to solve it. I will prepare the change.